lua/nvim_lsp/server_configs.lua

```diff
@@ -2,19 +2,19 @@ local util = require 'nvim_lsp/util'
 local api, validate, lsp = vim.api, vim.validate, vim.lsp
 local tbl_extend = vim.tbl_extend
 
-local skeleton = {}
+local server_configs = {}
 
 
-function skeleton.__newindex(t, template_name, template)
+function server_configs.__newindex(t, config_name, config_definition)
```

Comment:
just `config`

```suggestion
function server_configs.__newindex(t, config_name, config)
```

Reply:
This will introduce name collisions in the local functions [1](https://github.com/mjlbach/nvim-lsp/blob/b5ef0a67332d93c0ee95bbe3ad2dff7fab9ec301/lua/nvim_lsp/configs.lua#L40) and [2](https://github.com/mjlbach/nvim-lsp/blob/b5ef0a67332d93c0ee95bbe3ad2dff7fab9ec301/lua/nvim_lsp/configs.lua#L66).

I get this error when I rename: 
```
Error detected while processing FileType Autocommands for "lua":
E5108: Error executing lua ...ovim-unwrapped-master/share/nvim/runtime/lua/vim/lsp.lua:141: cmd type must be string or list.
```